cps/kni.c

```diff
@@ -798,8 +803,53 @@ data_ipv6_attr_cb(const struct nlattr *attr, void *data)
 	return MNL_CB_OK;
 }
 
+static void
+rd_send_err(const struct nlmsghdr *req, int err)
+{
+	struct nlmsghdr *rep;
+	struct nlmsgerr *errmsg;
+	char buf[MNL_SOCKET_BUFFER_SIZE];
+	struct sockaddr_nl rd_sa;
+	struct cps_config *cps_conf = get_cps_conf();
```

Comment:
Once `rd_cb()` receives `cps_conf` in the previous patch, we can pass `cps_conf` to `rd_send_err()` and drop `get_cps_conf()` above.